db/migrate/20151231160846_create_currencies.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCurrencies < ActiveRecord::Migration
+  def change
+    create_table :currencies do |t|
+      t.string :code
+      t.decimal :current_exchange_rate
+      t.datetime :exchange_rate_date
```

Comment:
We may want to do the exchange rate stuff a different way. There are gems. Leave out for now? Er, maybe we need it for data migration purposes?